mongodb-log.yml.example

```diff
@@ -0,0 +1,7 @@
+logs:
+# This sample file will forward mongodb error logs to NR once it's renamed to mongodb-log.yml
+# On Linux systems no restart is needed
+  - name: "mongodblog"
+    file: /var/log/mongodb/mongodb.log
+    attributes:
+      logtype: mongodb
```

Comment:
missing newline at the end of the file

Reply:
Done. Let me know if I missed anything else.